kopano/konnect/konnectd-identifier-registration.yaml

```diff
@@ -0,0 +1,33 @@
+---
+
```

Comment:
Remove empty line and fix indentation (remove 4 spaces at the beginning of each line)